packages/generators/add-generator.ts

```diff
@@ -161,7 +161,17 @@ export default class AddGenerator extends Generator {
 							this.configuration.config.webpackOptions.entry = entryOptions;
 							this.configuration.config.item = action;
 						});
-				}
+					} else {
+						if (action === "topScope") {
+							return this.prompt([
+								Input("topScope", "Enter line you want to add to topScope"),
```

Comment:
What about: "What do you want to add to topScope?"

Reply:
I agree on above